deploy/040-controller.yaml

```diff
@@ -54,6 +55,11 @@ spec:
                 configMapKeyRef:
                   name: driver-config
                   key: infraClusterLabels
+            - name: ALLOWED_STORAGE_CLASSES
+              valueFrom:
+                configMapKeyRef:
+                  name: driver-config
+                  key: allowedInfraStorageclasses
```

Comment:
there's another config that needs this `deploy/split-infra-tenant/040-controller.yaml`